server/api/socket/location.py

```diff
@@ -51,7 +52,24 @@
         room=sid,
         namespace="/planarally",
     )
-
+    sorted_initiatives = [
+        init.as_dict()
+        for init in Initiative.select()
+        .join(Shape, JOIN.LEFT_OUTER, on=(Initiative.uuid == Shape.uuid))
+        .join(Layer)
+        .where((Layer.location == location))
+        .order_by(Initiative.index)
+    ]    
+    await sio.emit(
+        "Markers.Set",
+        [
+            marker.as_dict()
+            for marker in Marker.select().where(Marker.user == user)
```

Comment:
This will return all the markers you own irregardless of the session/location you're in.

Reply:
And that's what you want, right?

Reply:
No you don't ..... I'll add them...

Reply:
There is still extra filtering required.  Imagine you have two completely different sessions in which you use markers.  This will return all markers from both sessions combined, which you wouldn't want.

Reply:
Now taken care of.